liftoff/1.6.3/README.md

```diff
@@ -0,0 +1,11 @@
+# Liftoff Container
+Main tool: [Liftoff](https://github.com/agshumate/Liftoff)
+
+Liftoff is a tool that accurately maps annotations in GFF or GTF between assemblies of the same, or closely-related species.
+
+# Example Usage
+```{bash}
+liftoff -g GCA_000146045.2_R64_genomic.gff.gz -o GCA_000146045.2_R64_to_GCA_000146045.2_advanced.gff3 -u unmapped_features_GCA_000146045.2_R64_advanced.txt -dir sandbox -chroms chroms.txt -unplaced unplaced.txt -copies GCA_000146045.2_R64_genomic.fna.gz
```

Comment:
This usage doesn't specify which python is being used, but the tests are using a specific version.

Reply:
This tool is ran by just calling the command 'liftoff'. In the dockerfile for my tests I use python3.8 because it is referencing a script and not the program to preform tests.